.github/ubiquibot-config.yml

```diff
@@ -7,3 +7,70 @@ command-settings:
 #  header:  "Thank you for contributing to UbiquiBot! Please be sure to set your wallet address before completing your first bounty so that the automatic payout upon task completion will work for you."
 #  helpMenu: true
 #  footer:  "###### Also please star this repository and [@ubiquity/devpool-directory](https://github.com/ubiquity/devpool-directory/) to show your support. It helps a lot!"
+
+incentives:
+  enabled: true
+  contentEvaluator:
+    enabled: true
+  userExtractor:
+    enabled: true
+    redeemTask: true
+  dataPurge:
+    enabled: true
+  formattingEvaluator:
+    enabled: true
+    scores:
+      br: 0
+      code: 1
+      p: 1
+      em: 0
+      img: 0
+      strong: 0
+      blockquote: 0
+      h1: 1
+      h2: 1
+      h3: 1
+      h4: 1
+      h5: 1
+      h6: 1
+      a: 1
+      li: 1
+      td: 1
+      hr: 0
+    multipliers:
+      - type: [ ISSUE, ISSUER, SPECIFICATION ]

```

Comment:
I'm still confused about this syntax. Can you explain what these are meant to express?

Reply:
Dynamic configurable replacement of
https://github.com/ubiquibot/comment-incentives/blob/525fcdccc8f0d9032a41cdaed3b4fbc6514d44b6/src/handlers/issue/comment-scoring-by-contribution-style.ts#L7

Reply:
If I understand correct, the syntax is: first value ISSUE / REVIEW , depends on location where the comment is located. Second value on the person: ISSUER /  ASSIGNEE / COLLABORATOR / CONTRIBUTOR . Third value depends on the action COMMENTED / SPECIFICATION / TASK (perhaps SPECIFIED / TASK_CREATED would be more clear but I think we can leave as is) . Therefore in general this is a triple [ location / person / action ] . Would be good to put this information into a readme or instruction somewhere.

Reply:
@gitcoindev you are correct, for now it is that way. But it is made so that we can extend it with more values if we need to. Any combination not found within the list defaults to a 0 multiplier to avoid accidents.